### ISSUE TITLE
Need to check if "style" prop is an array 

### ISSUE THREAD

**Issue by thomasttvo** [Ajackster/react-native-global-props#8]
```
this.props = { ...this.props, style: [customProps.style, this.props.style] };
```
when `this.props.style` is an array,  this will not work properly. In my case, it ignores customProps.style

**Comment by mahesh1996** [comment]
Yes. Need to check for both `customProps.style` and `this.props.style`.
Is anyone working on it?
I can pick this.

**Comment by thomasttvo** [comment]
@mahesh1996 while you're at it, can you take a look at https://github.com/Ajackster/react-native-global-props/issues/10 ? I suspect they're related. Thank you!

**Comment by mahesh1996** [comment]
@thomasttvo Tested on react-native 0.51. It works fine as `Text` can accept style object or recursive array of style objects.

**Comment by thomasttvo** [comment]
@mahesh1996  I got it on some elements too, but have deleted the code, will let you know if I happen to reproduce it.

**Comment by mahesh1996** [comment]
Also, make sure If you are using `native-base` in your project, then `Text` is imported from `react-native`.
